.circleci/config.yml

```diff
@@ -0,0 +1,207 @@
+    steps:
+      - run:
+          name: Early return if this build is from a forked PR
+          command: |
+            if [ -n "$CIRCLE_PR_NUMBER" ]; then
+              echo "Nothing to do for forked PRs, so marking this step successful"
+              circleci step halt
+            fi
+jobs:
+  checkout_code:
+    docker:
+      - image: circleci/clojure:lein-2.8.3
+    working_directory: ~/build
+    steps:
+      - checkout
+      - save_cache:
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - ~/build
+
+  bundle_dependencies:
+    docker:
+      - image: circleci/clojure:lein-2.8.3-node
+    working_directory: ~/build
+    steps:
+      - restore_cache:
+          keys:
+            - v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+            - v1-dependencies-{{ checksum "project.clj" }}
+            - v1-node-dependencies-{{ checksum "package.json" }}
+      - run: lein deps
+      - run: npm install
+      - save_cache:
+          key: v1-dependencies-{{ checksum "project.clj" }}
+          paths:
+            - ~/.m2
+      - save_cache:
+          key: v1-node-dependencies-{{ checksum "package.json" }}
+          paths:
+            - ~/node_modules
+
+  lint_code:
+    docker:
+      - image: circleci/clojure:lein-2.8.3
+    working_directory: ~/build
+    steps:
+      - restore_cache:
+          keys:
+            - v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+            - v1-dependencies-{{ checksum "project.clj" }}
+      - run: lein cljfmt check
+
+  verify_docs:
+    docker:
+      - image: circleci/clojure:lein-2.8.3-node
+    working_directory: ~/build
+    steps:
+      - restore_cache:
+          keys:
+            - v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+            - v1-dependencies-{{ checksum "project.clj" }}
+            - v1-node-dependencies-{{ checksum "package.json" }}
+      - run: ./.circleci/scripts/verify-docs.sh
+      - run: cd ./documentation && tar cvfz "../recide-docs.tgz" ./
+      - run: mkdir -p ./artifacts/docs
+      - run: mv ./recide-docs.tgz ./artifacts/docs/
+      - store_artifacts:
+          path: ./artifacts
+
+  verify_license_headers:
+    docker:
+      - image: circleci/clojure:lein-2.8.3
+    working_directory: ~/build
+    steps:
+      - restore_cache:
+          keys:
+            - v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+            - v1-dependencies-{{ checksum "project.clj" }}
+      - run: ./.circleci/scripts/verify-license-headers.sh "clj" "^;; Copyright 2016-[0-9]{4} Workiva Inc.$" project.clj profiles.clj
+      - run: ./.circleci/scripts/verify-license-headers.sh "java" "^// Copyright 2016-[0-9]{4} Workiva Inc.$"
+
+  unit_tests:
+    docker:
+      - image: circleci/clojure:lein-2.8.3
+    working_directory: ~/build
+    steps:
+      - restore_cache:
+          keys:
+            - v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+            - v1-dependencies-{{ checksum "project.clj" }}
+      - run: lein test
```

Comment:
I could not see unit tests step here https://circleci.com/gh/Workiva/recide/93?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link, what is the plan - switch from Workiva Build into CiCircle and use that step?